2023/day13/python - The DevOps Catalyst.md

```
@@ -0,0 +1,187 @@
+![Python: Versatile Programming Language for All](https://cdn.hashnode.com/res/hashnode/image/upload/v1686051137825/641f7d51-ee44-4ef5-abb3-182bbd91bb54.avif?w=1600&h=840&fit=crop&crop=entropy&auto=compress,format&format=webp)
+# Deep Dive in Git & GitHub for DevOps Engineers.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)